src/Message/RedirectPurchaseRequest.php

```diff
@@ -173,4 +179,17 @@ public function sendData($data) : ResponseInterface
 
         return new RedirectPurchaseResponse($this, $data);
     }
+
+    /**
+     * @param CreditCard $card
+     *
+     * @return string
+     * @throws NumberParseException
+     */
+    private function formattedPhoneNumber(CreditCard $card): string
+    {
+        $phoneUtil = PhoneNumberUtil::getInstance();
+        $numberProto = $phoneUtil->parse($card->getPhone(), $this->getCountry($card->getBillingCountry(), 'alpha2'));
```

Comment:
I am unsure whether to catch the exception from this and return a blank string or let it bubble up. I have no access to the sandbox at the moment so can't see which approach would be best